core/src/main/kotlin/Kotlin/KotlinLanguageService.kt

```diff
@@ -167,14 +167,14 @@ class KotlinLanguageService : CommonLanguageService() {
             return
         }
 
-        val nullabilityModifier = node.details(NodeKind.NullabilityModifier).singleOrNull()
+        val nullabilityModifier = node.detailOrNull(NodeKind.NullabilityModifier)
 
         if (node.isFunctionalType()) {
-            nullabilityModifier ?.apply {
+            nullabilityModifier ?.also {
```

Comment:
You can also use plain old `if`, nullabilityModifier will automatically cast to not-null type after check

Reply:
done